### ISSUE TITLE
General attributes

### ISSUE THREAD

**Issue by hgloeckner** [Geet-George/halodrops#125]
It might be nice to put campaign-specific attributes to the config rather than hard-coding them in the `processing.get_other_global_attributes()` function.

**Comment by Geet-George** [comment]
Yes, that was a temporary arrangement 😅 I agree, it should be made better, but how to add it to the config (or even if one should!) needs some discussion, I guess. 

That said, I would put this as low priority for during the campaign. This is mostly needed once data needs to be distributed publicly.

**Comment by hgloeckner** [comment]
we have filenames now as a function kwarg to the Sonde.get_l2_filename/Sonde.get_l3_filename (#135)
